### PR TITLE
Add getWorkPath()

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by tribal-tec** [comment]
Don't forget to unify with eq::Global::setWorkDir and co::Node::getWorkDir()
Name-wise: getCWD() or getWorkDir()?

**Comment by eile** [comment]
Ping - merge if happy.

File: lunchbox/file.h [hunk 1]
```diff
@@ -56,7 +56,15 @@ LUNCHBOX_API std::string getDirname( const std::string& filename );
...
+inline std::string getExecutablePath() { return getExecutableDir(); }
+
+/**
+ * @return the absolute path to the current working directory
+ * @version 1.14
+ */
+LUNCHBOX_API std::string getWorkDir();
 
 /**
  * Get the absolute path to the root path of the current executable.
```
Comment (tribal-tec) [inline lunchbox/file.h]: ..the root directory...

File: lunchbox/file.h [hunk 2]
```diff
@@ -56,7 +56,15 @@ LUNCHBOX_API std::string getDirname( const std::string& filename );
  * @return the absolute path to the current executable.
```
Comment (tribal-tec) [inline lunchbox/file.h]: path or dir now? I thought for executables it's still a path.

**Comment by tribal-tec** [comment]
Happy if comments are addressed, build is green and follow-up PRs for EQ and CO are in place :)

File: lunchbox/file.h (hunk 2 above)
Comment (eile) [inline lunchbox/file.h]: Boah ey! ;)

I though Path if if it includes the file, Dir if it is a directory?

**Comment by eile** [comment]
Eq and Co commits are ready - but this will not break build so can be merged right now?

File: lunchbox/file.h (hunk 2 above)
Comment (tribal-tec) [inline lunchbox/file.h]: Sorry, my bad. Thought it contains the executable name as well...

**Pull request merged**